addons/project/views/project_task_views.xml

```diff
@@ -9,7 +9,7 @@
                 <search string="Tasks">
                     <field name="name" string="Tasks" filter_domain="['|', ('name', 'ilike', self), ('id', 'ilike', self)]"/>
                     <field name="tag_ids"/>
-                    <field name="user_ids" filter_domain="[('user_ids.name', 'ilike', self), ('user_ids.active', 'in', [True, False])]"/>
```

Comment:
If the goal is to hide this field from project sharing, it should be moved to `view_task_search_form_project_fsm_base`

Reply:
Hello @MissingNoShiny , 
Thanks for the review, Done with the suggested changes... 
Please have a look once..
Thanks